.github/workflows/build-publish.yaml

```diff
@@ -3,113 +3,49 @@ name: Build and Publish Docker image
 on:
   pull_request:
     branches:
-    - main
-
+      - main
   push:
     branches:
-    - main
+      - main
     tags:
-    - v[0-9]+.[0-9]+.[0-9]+
+      - v[0-9]+.[0-9]+.[0-9]+
 
 jobs:
-  prepare:
+  build-and-publish:
     runs-on: ubuntu-latest
-    outputs:
-      FULL_IMAGE_TAG: ${{ steps.tag.outputs.tag }}
     steps:
-      - name: Set Tag
-        id: tag
-        run: |
-          export CI_COMMIT_SHORT_SHA=$(echo $GITHUB_SHA | cut -c1-8)
-          echo $CI_COMMIT_SHORT_SHA;
-          echo ${GITHUB_REF##*/};
-          echo $version_pattern;
-          if [[ ${GITHUB_REF} =~ $version_pattern ]]; then
-            echo "::set-output name=tag::${GITHUB_REF##*/}"
-          else
-            echo "::set-output name=tag::${GITHUB_REF##*/}-$CI_COMMIT_SHORT_SHA"
-          fi
-        env:
-          version_pattern: "tags\\/v[0-9]+\\.[0-9]+\\.[0-9]+"
-  build:
-    runs-on: ubuntu-latest
-    needs: prepare
-    environment:
-        name: cloudops
+      - name: Check out the repo
+        uses: actions/checkout@v3
 
-    steps:
-      - name: Docker image tag
-        env:
-          IMAGE_TAG: ${{ needs.prepare.outputs.FULL_IMAGE_TAG }}
+      - name: Build `version.json` file
         run: |
-          echo "Building an image with the following tag:"
-          echo $IMAGE_TAG
-
-      - name: Checkout
-        uses: actions/checkout@v2
-
-      - name: Login to Docker Hub
-        uses: docker/login-action@v1
-        with:
-          username: ${{ secrets.DOCKERHUB_USERNAME }}
-          password: ${{ secrets.DOCKERHUB_TOKEN }}
-
-      - name: Extract metadata (tags, labels) for Docker
-        uses: docker/metadata-action@v3
-        with:
-          images: $GITHUB_REPOSITORY
-
-      - name: Build Docker image
-        env:
-          IMAGE_TAG: ${{ needs.prepare.outputs.FULL_IMAGE_TAG }}
-        run: |
-          printf '{\n    "commit": "%s",\n    "version": "%s",\n    "image_tag": "%s",\n    "source": "%s",\n    "build": "%s"\n}\n' \
+          printf '{\n    "commit": "%s",\n    "version": "%s",\n    "source": "%s",\n    "build": "%s"\n}\n' \
             "$GITHUB_SHA" \
             "$GITHUB_REF" \
-            "$IMAGE_TAG" \
```

Comment:
I think we're safe to remove this since it's not a standard `version.json` field and the SHA is captured by the `commit` field. Happy to put in back though.